packages/connectivity_plus_linux/lib/src/network_manager.dart

```diff
@@ -31,7 +31,7 @@ class NetworkManager extends DBusRemoteObject {
           (value) => (value as DBusString).value,
           onError: (error) => print(error),
         )
-        .then((value) => value ?? fallback);
+        .then((value) => value);
```

Comment:
i think the problem is here since this returns null if dbus throws (dbus-daemon is not running in the ci)

Reply:
how about
```
.then((String? value) => value ?? fallback);
```

Reply:
nice catch!

Reply:
Btw, I was able to reproduce the problem in a Docker container that doesn't have a D-BUS daemon running, similarly to the CI environment.